### PR TITLE
Today's To-do app - Pull Request

### PR DESCRIPTION
**Pull request by AmaduKamara**
# Today's To-do app

 Completed the following given in the requirements of the project -  **To-Do list: list structure**:
- Set up a new project with Webpack
- Set up GitHub workflow and linter configurations
- Create HTML structure of the app
- Styled the UI based on the existing website design in the project requirement
- Created an array with list of todo objects with:
   ☑️ description [string].
   ☑️ completed [bool].
   ☑️ index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- All your source files (index.html, index.js, and style.css) are located in /src, and on `npm run build`, files will be generated by Webpack and served by the Webpack dev server from the /dist folder.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @AmaduKamara 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/main.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #d8d8d8;
+  padding: 15px 0;
+  padding: 15px;
+}
+
+.check-div {
+  /* stylelint-disable-next-line csstree/validator */
```
Comment (mikenath223) [inline src/main.css]: - [x] Kindly remove this stylelint disable line its best to fix the issue in the code.

Comment (AmaduKamara) [reply src/main.css]: Sure 👍It's fixed in the CSS file now. Thanks

**Comment by omar-labana** [comment]
Hi @AmaduKamara, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**